website/docs/developer-guide/chaoscenter-developer-guide.md

```diff
@@ -0,0 +1,219 @@
+---
+id: chaoscenter-developer-guide
+title: ChaosCenter Developer Guide
+sidebar_label: ChaosCenter Developer Guide 
+---
+
+---
+
+## **Prerequisites**
+:::note
+This document is ietented to be implemented locally. Please do not use in dev or prod environmentss
+:::
+
+- Kubernetes 1.17 or later
+- Helm3 or Kubectl
+- Node and npm
+- Docker
+- Golang
+- Local Kubernetes Cluster (via minikube, k3s or kind)
+
+## **Control Plane**
+Backend components consist of three microservices
+1. GraphQL-Server
```

Comment:
```suggestion
1. Backend server
```

Reply:
I think it's fine to have graphql server since we have 2 backend servers

Reply:
Shoud we use graphql server?? 
in [docs](https://docs.litmuschaos.io/docs/architecture/architecture-summary) using graphql